nautobot/ipam/tests/test_views.py

```diff
@@ -208,7 +208,7 @@ def test_list_objects_with_permission(self):
                         response, f'<a href="{locations_list_url}?prefixes={prefix.pk}" class="badge">{count}</a>'
                     )
                 elif count == 1:
-                    self.assertBodyContains(response, hyperlinked_object(prefix.locations.first()))
+                    self.assertBodyContains(response, hyperlinked_object(prefix.locations.first().name))
```

Comment:
I think this'd be more correct?

```suggestion
                    self.assertBodyContains(response, hyperlinked_object(prefix.locations.first(), "name"))
```

Reply:
Yep good catch.  Otherwise it would incorrectly pass if the name was anywhere in the field along with other text rather than just the pure `location.name`

Reply:
Yep good catch. Otherwise it would incorrectly pass if the name was anywhere in the field along with other text rather than just the pure `location.name`